src/site/xdoc/checks/annotation/annotationlocation.xml

```diff
@@ -12,10 +12,10 @@
       <subsection name="Description" id="Description">
         <div>
           Checks location of annotation on language elements.
-          By default, Check enforce to locate annotations immediately after
```

Comment:
Please look at IDE file that is  sibling to this file, it is named template.
You need to add macros to template.
Looks at example commit that I referenced.